From 7fcf1287c967be655c24e3260ed2fe41a628272a Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 21 Jul 2020 12:45:52 +0200
Subject: [PATCH 1/1] dfu: DFU_MTD depends on CMD_MTDPARTS

Function mtdparts_init() is needed for the DFU MTD driver.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 drivers/dfu/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/dfu/Kconfig b/drivers/dfu/Kconfig
index 46c915f9e5..106f38817a 100644
--- a/drivers/dfu/Kconfig
+++ b/drivers/dfu/Kconfig
@@ -70,6 +70,7 @@ config DFU_SF_PART
 config DFU_MTD
 	bool "MTD back end for DFU"
 	depends on DM_MTD
+	depends on CMD_MTDPARTS
 	help
 	  This option enables using DFU to read and write to on any MTD device.
 
-- 
2.27.0

